policy/_db/migrations/20150825180747_InitialSchema.sql

```diff
@@ -0,0 +1,15 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `ruleList` (
```

Comment:
I don't think we closed this out.  I also prefer that they be separate, more for architectural cleanness than for security -- there should be separate methods to ask about blacklist, whitelist, and PSL (future), so it seems cleaner to have one table per method.